packages/server/.vscode/launch.json

```diff
@@ -54,6 +54,19 @@
       "env": {
         "NODE_ENV": "development"
       }
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Message Relay",
+      "cwd": "${workspaceRoot}",
+      "protocol": "inspector",
+      "internalConsoleOptions": "neverOpen",
+      "runtimeArgs": ["-r", "ts-node/register"],
+      "args": ["${workspaceFolder}/src/message/firebase-relay.ts"],
```

Comment:
What's the benefit of this being a VSCode debug command?

Reply:
`firebase-relay is run as a separate process. For debugging, it's nice to start the process through vscode for breakpoint setting, etc.